controllers/oauth.go

```diff
@@ -80,6 +81,34 @@ func OAuth(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	// Check if the user is a organiser
+
+	isOrganiser := utils.CheckUserOrgs(accessToken, userInfo.Username)
+	if isOrganiser {
+
+		jwtString, err := utils.GenerateLoginJwtString(utils.LoginJwtFields{
+			Username: userInfo.Username,
+			UserType: OAUTH_TYPE_ORGANISER,
+		})
+
+		if err != nil {
+			utils.LogErrAndRespond(r, w, err, "Error generating a JWT string.", http.StatusInternalServerError)
+			return
+		}
+		resFields := OAuthResBodyFields{
+			Username:  userInfo.Username,
+			Name:      userInfo.Name,
+			Email:     userInfo.Email,
+			College:   "",
+			Type:      OAUTH_TYPE_ORGANISER,
+			IsNewUser: false,
```

Comment:
Does this allow KOSS members to also register projects?

Reply:
No this dosent allow KOSS Executives to register projects

Reply:
Now allows KOSS Executives to be mentors